Lesson23/assets/js/third.js

```diff
@@ -0,0 +1,14 @@
+
+// Request a number and display all the divisors of it. For example, for number 8 it would be 2, 4, and 8.
+
+let num = prompt('Enter a number');
+
+let i=1;
+let  k = `d=`;
+
+for (let i=1; i<=num; i++) {
+    if (num%i==0){
```

Comment:
u will also add the number too

Reply:
> u will also add the number too

If you mean that I display the number we checked - that was in the assignment Or something else?